projects/elife.yaml

```diff
@@ -369,7 +369,7 @@ api-gateway:
                 gcslogging:
                     bucket: end2end-elife-fastly
                     path: api-gateway--test/
-                    period: 1800
+                    period: 900
```

Comment:
this is just to exemplify what would happen when running `update_infrastructure:api-gateway--delta`